content/2015.11.shweta-oak.rst

```diff
@@ -0,0 +1,80 @@
+Discovery of Kinto servers
+###########################
+
+:slug: outreachy-2016
+:date: 2015-11-27
+:lang: en
+:url: shweta-oak-kinto-2016
+:summary:
+    Work on Kinto project - making instances discoverable, during internship.
```

Comment:
And then the summary would be somethink like "Where we discuss how to make kinto instances discoverable".